app/scripts/modules/kubernetes/src/pipeline/stages/runJob/runJobStage.html

```diff
@@ -18,6 +18,20 @@
     </namespace-select-field>
   </stage-config-field>
 
+  <stage-config-field label="Image">
+    <ui-select name="containerSelect" ng-model="ctrl.stage.container" class="form-control input-sm" required>
+        <ui-select-match>{{ ctrl.stage.container.imageDescription | kubernetesImageId }}</ui-select-match>
+        <ui-select-choices
+          repeat="container in ctrl.containers | orderBy:'imageDescription.imageId'"
```

Comment:
Hmm really didn't see this `orderBy` when I made that `imageId` suggestion. What order do they end up in? The ones that come from triggers/pipeline context should be sorted to the top.

Reply:
Maybe that's completely determined by the `group-by` function. In that case, omit this line.

Reply:
Yeah, they end up in a weird order. There's no sorting logic in the `group-by` function either so I need to figure out another solution.